packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -162,8 +162,14 @@ export enum PluginExtensionPoints {
   DataSourceConfig = 'grafana/datasources/config',
   ExploreToolbarAction = 'grafana/explore/toolbar/action',
   UserProfileTab = 'grafana/user/profile/tab',
+  QueryToAppPlugin = 'grafana/query/query-to-app-plugin',
```

Comment:
Currently extensions is only supported for app plugins, but that may change in the future. I suggest to change this to grafana/query/row-header-actions or something like that.